src/main/java/pages/cartPage/ShoppingCartTable.java

```diff
@@ -0,0 +1,282 @@
+package pages.cartPage;
+
+import enums.Currency;
+import lombok.Getter;
+import lombok.Setter;
+import models.ProductModel;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import static engine.drivers.WebDriverFactory.getWebDriverInstance;
+
+@Getter
+@Setter
+public class ShoppingCartTable extends BaseTable {
```

Comment:
If you want to extend base table here I will strongly recommend using composition inside baseTable for BasePage:)

Reply:
Change of inheritance  to compositions.
Now ```baseTable``` is field for ```ShoppingCartTableRow``` and ```ShoppingCartTable``` inherits from ```ShoppingCartTableRow```